Tests/Invoke-PxGet.Tests.ps1

```diff
@@ -81,103 +71,124 @@ BeforeAll {
 
 Describe 'Invoke-Pxget' {
     BeforeEach { 
-        Init
-        New-TestRoot 
+        $script:testRoot = $null
+        $script:moduleList = @()
+        $script:failed = $false
+        $Global:Error.Clear()
+        $script:testRoot = Join-Path -Path $TestDrive -ChildPath ($script:testNum++)
+        New-Item -Path $script:testRoot -ItemType 'Directory'
     }
     AfterEach { 
-        Reset 
+        if( Test-Path -Path "$testRoot \pxget.json" )
+        {
+            Remove-Item -Path "$testRoot \pxget.json"
+        }
     }
 
     It 'should pass when there is a valid module in the pxget file' {
         $contents = @"
         {
             "PSModules": [
                 {
-                    "Name": "PackageManagement",
-                    "Version": "1.4.7"
+                    "Name": "NoOp",
+                    "Version": "1.0.0"
                 }
             ]
         }
 "@
         GivenPxGetFile -Contents $contents
         WhenInvokingPxGet
         ThenSucceeded
+        ThenInstalled -ModuleName 'NoOp' -Version '1.0.0'
     }
 
     It 'should pass when the module to be installed already exists' {
         $contents = @"
         {
             "PSModules": [
                 {
-                    "Name": "PackageManagement",
-                    "Version": "1.4.7"
+                    "Name": "NoOp",
+                    "Version": "1.0.0"
                 }
             ]
         }
 "@
         GivenPxGetFile -Contents $contents
         WhenInvokingPxGet
         ThenSucceeded
+        ThenInstalled -ModuleName 'NoOp' -Version '1.0.0'
     }
 
     It 'should pass when the module to be installed is a prerelease version' {
         $contents = @"
         {
             "PSModules": [
                 {
-                    "Name": "PackageManagement",
-                    "Version": "1.2.0-preview"
+                    "Name": "NoOp",
+                    "Version": "1.0.0-alpha26"
                 }
             ]
         }
 "@
         GivenPxGetFile -Contents $contents
         WhenInvokingPxGet
         ThenSucceeded
+        ThenInstalled -ModuleName 'NoOp' -Version '1.0.0'
     }
 
     It 'should pass when there are multiple modules to be installed' {

```

Comment:
Seeing this test fixture made me realize we need a test to ensure that we can install the same module, but with different versions. Please add that test.

Reply:
Had to do this with Carbon instead of NoOp.  I tried NoOp with Versions 1.0.0 and 1.0.1.  Version 1.0.1 of NoOp was listed in WhsPowerShell Repository and I think this is what caused it to fail on build, even though it worked when I ran it locally.  I switched to Carbon versions 2.11.0 and 2.10.0 both of which were listed under powershellgallery and this worked on AppVeyor builds.